### PR TITLE
Use main branch for storking new apps

### PR DESCRIPTION
**Pull request by GiudGiud**
Came up live in a tutorial today :( 
refs #20124

### CODE REVIEW

**Review by cticenhour — CHANGES_REQUESTED** [review]
Should reference #16883 instead. I invalidated all of the cancelled CIVET recipes before I noticed this, so while making this change in your commit message, might want to rebase on current devel.

File: scripts/stork.sh [hunk 1]
```diff
@@ -124,7 +124,7 @@ if [[ "$kind" == "app" ]]; then
     cp $MOOSE_DIR/.gitignore $dir/
 
     dir="$PWD/$dir"
-    (cd $dir && git init && git add * .clang-format .gitignore && git commit -m"Initial files")
+    (cd $dir && git init -b main && git add * .clang-format .gitignore && git commit -m"Initial files")
```
Comment (cticenhour) [inline scripts/stork.sh:127]:
```suggestion
    (cd $dir && git init && git branch -m main && git add * .clang-format .gitignore && git commit -m"Initial files")
```
This change is more backwards compatible with `git < 2.28`, which did not have a `-b` flag for `git-init`.

**Review by cticenhour — CHANGES_REQUESTED** [review]

**Comment by GiudGiud** [comment]
thanks for the fix!
Changed the issue in the commit and rebased on devel. Should be good now

**Comment by GiudGiud** [comment]
@cticenhour

**Comment by cticenhour** [comment]
Still seeing the prior issue number in the commit message. Could you amend this again? Otherwise this looks good.

**Comment by GiudGiud** [comment]
weird I really thought I had fixed that. Must have dreamed it.
should be good now

**Review by cticenhour — APPROVED** [review]

**Pull request merged**